#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_mytharchive_ubuntu_directories.dpatch by laga <>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change some paths to match Debian/Ubuntu better

@DPATCH@

--- a/mytharchive/mytharchive/archivesettings.cpp.old	2008-04-13 18:47:22.991881577 +0200
+++ b/mytharchive/mytharchive/archivesettings.cpp	2008-04-13 18:47:45.431286036 +0200
@@ -15,7 +15,7 @@
 {
     HostLineEdit *gc = new HostLineEdit("MythArchiveTempDir");
     gc->setLabel(QObject::tr("Myth Archive Temp Directory"));
-    gc->setValue("");
+    gc->setValue("/var/lib/mytharchive/temp/");
     gc->setHelpText(QObject::tr("Location where MythArchive should create its "
             "temporory work files. LOTS of free space required here."));
     return gc;
